app-libs/sgx-runtime/src/lib.rs

```diff
@@ -390,6 +436,8 @@ construct_runtime!(
 
 		Notes: pallet_notes::{Pallet, Call, Storage} = 40,
 		SessionProxy: pallet_session_proxy::{Pallet, Call, Storage} = 41,
+
+		Assets: pallet_assets::{Pallet, Call, Storage, Event<T>} = 50,
```

Comment:
Why do we have 2 different indexes here?

Reply:
omg